checkpoint/recovery.go

```diff
@@ -170,7 +187,10 @@ func RecoverWithDb(
 		return nil, fmt.Errorf("remove old bootstrap data: %w", err)
 	}
 	logger.Info("recover from uri", zap.String("uri", cfg.Uri))
-	cpFile, err := copyToLocalFile(ctx, logger, fs, cfg.DataDir, cfg.Uri, cfg.Restore)
+	cpFile, err := copyToLocalFile(
+		ctx, logger, fs, cfg.DataDir, cfg.Uri, cfg.Restore,
+		cfg.RetryMax, cfg.RetryDelay,
+	)
```

Comment:
at that point it probably makes sense to change the function signature to

```suggestion
	cpFile, err := copyToLocalFile(ctx, logger, cfg, fs)
```

Wdyt?

Reply:
Fixed, thanks